config/services/redis-server.service.d/restart.conf

```diff
@@ -0,0 +1,4 @@
+[Service]
+# Always restart redis in case it dies for some reason
+Restart=always
```

Comment:
Why do you need this file as well? This key is already defined in the service file

Reply:
I wasn't paying attention to this key, it's the one below I was more interested in, I'll fix it